drivers/gles3/rasterizer_gles3.cpp

```diff
@@ -251,21 +255,16 @@ void RasterizerGLES3::initialize() {
 #endif // GLES_OVER_GL
 #endif // CAN_DEBUG
 
-	print_line("OpenGL Renderer: " + RS::get_singleton()->get_video_adapter_name());
-	storage.initialize();
-	canvas.initialize();
-	//	scene.initialize();
-
-	// make sure the OS knows to only access the renderer from the main thread
-	OS::get_singleton()->set_render_main_thread_mode(OS::RENDER_MAIN_THREAD_ONLY);
-}
-
-RasterizerGLES3::RasterizerGLES3() {
-	canvas.storage = &storage;
-	canvas.scene_render = &scene;
-	//storage.canvas = &canvas;
-	//scene.storage = &storage;
-	//storage.scene = &scene;
+	// OpenGL needs to be initialized before initializing the Rasterizers
+	config = memnew(GLES3::Config);
+	texture_storage = memnew(GLES3::TextureStorage);
+	material_storage = memnew(GLES3::MaterialStorage);
+	mesh_storage = memnew(GLES3::MeshStorage);
+	particles_storage = memnew(GLES3::ParticlesStorage);
+	light_storage = memnew(GLES3::LightStorage);
+	storage = memnew(RasterizerStorageGLES3);
+	canvas = memnew(RasterizerCanvasGLES3(storage));
+	scene = memnew(RasterizerSceneGLES3);
 }
```

Comment:
Now these are dynamically allocated, is there a `memdelete` for them?

Reply:
Of course not!

Reply:
Of course not!

...added now.